flaps/flaps.go

```diff
@@ -203,9 +204,18 @@ waiting:
 	}
 }
 
+func snakeCase(s string) string {
+	pat := regexp.MustCompile("[A-Z]")
```

Comment:
not sure if Go optimizes it somehow but this is better placed as toplevel var so it is compiled once.

Reply:
I think it won't and I was lazy. Updated in the last revision.